src/themes/custom_theme/custom_theme.html

```diff
@@ -0,0 +1,74 @@
+<div class="custom_theme" title="Customize appearance of chart">
+	<div id="properties">
+			<table width="100%">
+				<tr>
+					<td><strong>Chart background</strong></td>
+					<td>
+					<input value="" id="temp_box" type="text"/>
```

Comment:
@arnabk had to add this ("not visible") text box in order to remove focus off colorpicker. Tried using `blur` function but it didn't work because the colorpicker popup can't be closed using that.